### ISSUE TITLE
Pages don't display on mobiles

### ISSUE THREAD

**Issue by yangfuzhang** [vuematerial/vue-material#1137]
I'm using the beta version of vue-material, everything works fine on PC, but pages could not be displayed on mobiles. And I also find that the vuematerial.io official website has the same problem.

**Comment by marcosmoura** [comment]
Can you send me a screenshot or something? Did something appeared in devtools console?

**Comment by yangfuzhang** [comment]
The devtools did show some error about <md-field>, but I guess that's not why the website could not be displayed on mobile phone, since it worked well on PC.

The pages are totally empty on mobile. Here's some screenshots for your reference.

**This is what happen to the official website:**
![_20171115004206](https://user-images.githubusercontent.com/13726091/32792488-6b1c086a-c99e-11e7-9b19-79051916cdf1.png)

**And this is what happened to my website:**
![_20171115004156](https://user-images.githubusercontent.com/13726091/32792567-96621c9e-c99e-11e7-9844-8cc9c3edf51c.png)

**Comment by marcosmoura** [comment]
Which browser?

**Comment by yangfuzhang** [comment]
I tried every browser I install on my Android mobile, including UC browser, all the same problem.  Does it happen to you (about the vuematerial.io website on mobile phone)?
By the way I changed to the beta version today, and the old version displayed fine on mobile.

**Comment by marcosmoura** [comment]
Fir me it's working fine. Google Chrome. Android 8

**Comment by yangfuzhang** [comment]
Finally figure out the problem, some browsers don't support Object.values() function.

**Comment by timoschwarzer** [comment]
Which browser did you try specifically?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values#Browser_compatibility

**Comment by marcosmoura** [comment]
I will provide a polyfill for this.

**Comment by Tahywoh** [comment]
Yes o, same here. My vue web app displays blank on mobile, all mobile android browsers, except the latest version of chrome and firefox, please how do I resolve this?
